wave_geometry/include/wave/geometry/rotation.hpp

```diff
@@ -57,6 +57,9 @@ class Rotation {
     // Operator overloads.
     Rotation operator*(const Rotation &R) const;
     friend std::ostream &operator<<(std::ostream &stream, const Rotation &R);
+
+    // Static functions.
+    static bool isValidRotationMatrix(const Mat3 &input_matrix);
```

Comment:
This should not be a member function, since it does not use any non-public members of `Rotation` (or _any_ members of `Rotation`, for that matter).

(see http://www.drdobbs.com/cpp/how-non-member-functions-improve-encapsu/184401197)